### ISSUE TITLE
Idea: auto-assign workers to new jobs

### ISSUE THREAD

**Issue by lxsli** [pmotschmann/Evolve#402]
When I buy a new cement factory, or anything else, it'd be great to get workers auto-assigned from the default job. This would save a great deal of tab-switching.

Foundries are a little tricky as there are multiple options. You could mirror the default job system for crafting, this would work well imo. Or hardcode brick (available for all races afaik?), or simply not auto-assign crafters for now and make that a separate piece of work.

This should be default behaviour not an option imo as I believe buy-job-but-don't-assign should always be less frequent than buy-and-assign, so the user is always saving clicks with the new behaviour. I don't know of any achievements this would mess with?

**Comment by boozledorf** [comment]
Disagree on default behavior.  When I run 4* runs, I almost never want all of my jobs auto-assigned, as I'll very quickly run out of food production.  I always leave either my bankers or priests empty -- sometimes both -- until Apartments.

**Comment by lxsli** [comment]
Yeah I do the same. Isn't it easier to have to unassign those jobs, than to have to assign every other job, though?

I'm happy for this to be an option if necessary, it's just good software design to try not to bloat the config too much.

**Comment by lxsli** [comment]
As a bit of a tangent, it'd be helpful to be able to use any job, priests specifically, as the default job. I guess it could overflow into unemployed/hunter?

**Comment by lxsli** [comment]
One way to do this would be to allow jobs to be reordered, and new citizens enter the first open job (from the top, say). Maybe some drag n drop? You could leave Unemployed on top if that's really where you want new citizens to go, or move it to the bottom to have it as a last resort.

**Comment by lxsli** [comment]
I usually run with lumberjacks as my default job btw; @boozledorf do you usually run with farmer? That would explain your issue.
